### ISSUE TITLE
delaunay moved in matplotlib 2.1

### ISSUE THREAD

**Issue by thomasaarholt** [delmic/odemis#443]
As a heads up, the delaunay module is moved in matplotlib 2.1 and the Triangulation function is now under matplotlib.tri. I could not figure out where DuplicatePointWarning, which is also imported in odemis, has moved.

```
---------------------------------------------------------------
ImportError                   Traceback (most recent call last)
<ipython-input-9-e51787e11528> in <module>()
----> 1 from odemis.acq import align
      2 f = align.AutoFocus(sed, ebeam, ebeam_focus)

/usr/lib/python2.7/dist-packages/odemis/acq/__init__.py in <module>()
     34 from odemis import model
     35 from odemis.acq import _futures
---> 36 from odemis.acq.stream import FluoStream, SEMCCDMDStream, \
     37     OverlayStream, OpticalStream, EMStream, SEMMDStream
     38 from odemis.util import img, fluo

/usr/lib/python2.7/dist-packages/odemis/acq/stream/__init__.py in <module>()
     31 from ._helper import *
     32 from ._live import *
---> 33 from ._static import *
     34 from ._sync import *
     35 

/usr/lib/python2.7/dist-packages/odemis/acq/stream/_static.py in <module>()
     33 from odemis.acq import calibration
     34 from odemis.model import MD_POS, MD_PIXEL_SIZE, VigilantAttribute
---> 35 from odemis.util import img, conversion, polar, spectrum
     36 from scipy import ndimage
     37 

/usr/lib/python2.7/dist-packages/odemis/util/polar.py in <module>()
     18 
     19 import math
---> 20 from matplotlib.delaunay import Triangulation
     21 from matplotlib.delaunay.triangulate import DuplicatePointWarning
     22 from numpy import ma

ImportError: No module named delaunay
```

**Comment by pieleric** [comment]
Yes, we are aware of it. Unfortunately it's not a "move" it's a replacement:
http://matplotlib.org/api/api_changes.html?highlight=delaunay#matplotlib-delaunay

We've tried using the new functions, but they are a lot slower, so we were not very eager to do so. You can see it around l. 124 of src/odemis/util/polar.py . If you disable the code around that line, and enable the next block of code, you can test with the new function.

**Comment by thomasaarholt** [comment]
In case you're interested, the [`scipy.spatial.Delaunay`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.Delaunay.html) modeule uses [qHull ](http://www.qhull.org/)instead of matplotlib's fauly implementation.

**Comment by pieleric** [comment]
Fixed in commit 38eea4dcf1e62fac4862e91ac70a58b3b277e0cc.
